### DISCUSSION TITLE
Register input for borderless window

### DISCUSSION THREAD

**Discussion by ben-forster** [godotengine/godot-proposals#6464] [GUI]
When enabling the borderless window setting for your project, for input to be registered, you need to click inside the window first. This, compared to disabling the setting, means the engine registers input without having to click it. This should be changed.

**Comment by Zireael07** [comment]
Yes it's a bug, I saw someone report it.

Please use other channels for questions like this (eg. Discord, forums) because this is for code-less proposals and ideas, not all manners of questions

**Reply by ben-forster** [reply]
Thank you for the reply.

I was going to write up a proposal if this was intended behaviour. May I have a link to the report if you can find it?

**Reply by Zireael07** [reply]
I was thinking of this one #74286 but now I'm not 100% sure if it's the same - anyway looks like a bug, report it on the main repo instead

**Reply by ben-forster** [reply]
I'll look, cheers!
